index.js

```diff
@@ -0,0 +1,74 @@
+import luxon from './modules/luxon.js';
+import AwesomeBooks from './modules/books.js';
+
+const awesomeBooks = new AwesomeBooks();
+awesomeBooks.load();
+if (awesomeBooks.getBooks().length < 1) {
+  awesomeBooks.addBook('Double Standard 2', 'Dr. Samsul Arefin');
+}
+
+const bookListSection = document.querySelector('#book-list');
+
+const renderBookList = () => {
+  bookListSection.innerHTML = awesomeBooks
+    .getBooks()
+    .map(
+      (book, index) => `
+        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
+            <div>
+                <p class="title">"${book.title}" by ${book.author}</p>
+            </div>
+            <button data-id=${book.id} class="remove">Remove</button>
+        </article>`,
+    )
+    .join('');
+};
+renderBookList();
+
```

Comment:
- [ ] [OPTIONAL] It is usually best practice to break your code into as much modules as you can. In this case, some of the functions in this file can be extracted into separate files and imported into this.

Reply:
Noted.
Thanks for your suggestion. I'll implement this way.